src/components/MainNav/CurrentApp/CurrentApp.js

```diff
@@ -4,22 +4,45 @@
 
 import React from 'react';
 import PropTypes from 'prop-types';
-import NavButton from '../NavButton';
+import AppIcon from '@folio/stripes-components/lib/AppIcon';
+import Badge from '@folio/stripes-components/lib/Badge';
+import Headline from '@folio/stripes-components/lib/Headline';
+import css from './CurrentApp.css';
 
 const propTypes = {
-  currentApp: PropTypes.object,
+  currentApp: PropTypes.shape(
+    {
+      displayName: PropTypes.string,
+      description: PropTypes.string,
+    },
+  ),
+  id: PropTypes.string,
+  icon: PropTypes.oneOfType([
+    PropTypes.element,
+  ]),
+  badge: PropTypes.oneOfType([
+    PropTypes.string,
+    PropTypes.number,
+  ]),
 };
 
-const CurrentApp = ({ currentApp }) => {
-  const { displayName, description } = currentApp;
+const defaultProps = {
```

Comment:
Instead of having "Folio" as default it should be the last open app as default (or the first app in the list) meaning that you shouldn't be able to _not_ have any open apps.

I think we can leave it like this for now since the "last open app" feature isn't made yet. The last open app should be stored somewhere - probably on each individual user. It probably needs it's own issue I assume.